src/App.tsx

```diff
@@ -39,11 +47,13 @@ export default class App extends React.Component {
                         resizeMode={"contain"}
                         source={require("./assets/react-logo.png")}
                     />
-                    <Text style={styles.appTitle}>Welcome to React Native Web️</Text>
-                    <Text style={styles.appSubtitle}>Vanilla Edition</Text>
+                    <Text style={styles.appTitle}>Welcome to React Native Web</Text>
+                    <Text style={styles.appSubtitle}>Typescript Edition</Text>
                 </View>
                 <View style={{ alignItems: "center", flex: 3 }}>
-                    <Text style={styles.appIntro}>To get started, edit src/App.js and save to reload.</Text>
+                    <Text style={styles.appIntro}>
+                        To get started, run "npm run watch", then edit src/App.tsx and save to reload.
```

Comment:
Would recommend this be changed to `yarn watch` I tried to avoid using NPM due to known issues on [CRA](https://github.com/react-community/create-react-native-app/issues/233) and [CRNA](https://github.com/react-community/create-react-native-app/issues/229) I know that the scripts are independant of packages but generally good to stay consistent